.github/stale.yml

```diff
@@ -0,0 +1,37 @@
+# Default values: https://probot.github.io/apps/stale/#installation
+
+daysUntilStale: 180
+
+exemptLabels:
+  - C-tracking-issue      # keep tracking issues open
+  - "Feature accepted"    # keep accepted features
+
+staleLabel: stale
+
+markComment: >
+  As there hasn't been any recent activity here for 180 days this has been
+  marked as stale, and if no further activity happens for 7 days this will be
+  automatically closed. This is an automatic action and may be in error, if this
+  issue should remain open please comment to that effect!
+
+  When commenting, we'd be eternally grateful if some details such as these
+  could be included:
+
+  * Is this still relevant?
+  * If so, what is blocking it?
+  * Is it known what could be done to help move this forward?
+
+  Thank you for contributing!
+
+  If you're reading this comment from the distant future, fear not if this issue
+  was closed automatically. If you believe it's still an issue please leave a
+  comment and a team member can reopen this issue. Opening a new issue is also
+  acceptable!
+
+closeComment: >
+  A comment to confirm that this is being closed due to being stale and the
```

Comment:
I am going to close this for now as I did not see an update. If it's still an issue, I am sorry. Please leave a comment and a team member can reopen it, or open a new issue to start a new discussion of the problem.